openshift-serverless-clients.spec

```
@@ -6,6 +6,7 @@
 %global kn_version 0.26.0
 %global kn_release 1
 %global kn_cli_version v%{kn_version}
+%global kn_event_image sender-nvr
```

```suggestion
%global kn_event_image %{nil}
```

https://stackoverflow.com/a/3919298/844449

---

@Kaustubh-pande what does the tooling say? Would it be able to work with `%{nil}`?